lib/internal/streams/readable.js

```diff
@@ -822,9 +822,7 @@ Readable.prototype.pipe = function(dest, pipeOpts) {
   // Start the flow if it hasn't been started already.
 
   if (dest.writableNeedDrain === true) {
-    if (state.flowing) {
-      pause();
-    }
+    pause();
```

Comment:
I don't actually remember why we had this condition...

Reply:
Ah, great – I was just looking into it, and this seems to be a better fix.

Reply:
I expect it was just a slight optimization from when `src.pause()` was called directly.